File: src/main/java/com/github/show0611/showPlugins/mserver/utilities/SPMSData.java
package com.github.show0611.showPlugins.mserver.utilities;

import com.github.show0611.showPlugins.mserver.Main;
import org.bukkit.configuration.file.FileConfiguration;

import java.util.ArrayList;
import java.util.LinkedHashMap;
import java.util.List;

/**
 * Created by show0611 on 2017/01/29.
 */
public class SPMSData {
    public static List<String> Muters = new ArrayList<>();
    public static List<String> HideMuters = new ArrayList<>();
    public static List<String> ShowMuters = new ArrayList<>();
    public static List<String> PermPlayers = new ArrayList<>();
    public static FileConfiguration conf = Main.main.getConfig();
    public static LinkedHashMap<String, LinkedHashMap<String, String>> PlayerData = new LinkedHashMap<>();
    public static LinkedHashMap<String, LinkedHashMap<String, String>> BannedPlayers = new LinkedHashMap<>();
    public static LinkedHashMap<String, List<LinkedHashMap<String, Object>>> HomeData = new LinkedHashMap<>();
}

File: src/main/java/com/github/show0611/showPlugins/mserver/utilities/SPMSData.kt
package com.github.show0611.showPlugins.mserver.utilities

import com.github.show0611.showPlugins.mserver.Main

import java.util.ArrayList
import java.util.LinkedHashMap

/**
 * Created by show0611 on 2017/01/29.
 */
object SPMSData {
    var Muters: MutableList<String> = ArrayList()
    var HideMuters: MutableList<String> = ArrayList()
    var ShowMuters: MutableList<String> = ArrayList()
    var PermPlayers: MutableList<String> = ArrayList()
    var conf = Main.main.config
    var PlayerData = LinkedHashMap<String, LinkedHashMap<String, String>>()
    var BannedPlayers = LinkedHashMap<String, LinkedHashMap<String, String>>()
    var HomeData = LinkedHashMap<String, MutableList<LinkedHashMap<String, Any>>>()
    var Memory = mutableListOf<String>()
}
